pkg/driver/controller.go

```diff
@@ -323,11 +323,11 @@ func getAccessibilityRequirements(requirement *csi.TopologyRequirement) string {
 		return ""
 	}
 
-	preferredNode, exists := requirement.GetPreferred()[0].GetSegments()[HostTopologyKey]
```

Comment:
I don't see that variable HostTopologyKey has been removed?

Reply:
removed